auto_nag/bzcleaner.py

```diff
@@ -623,11 +623,11 @@ def get_args_parser(self):
         """Get the argumends from the command line"""
         parser = argparse.ArgumentParser(description=self.description())
         parser.add_argument(
-            "-d",
-            "--dryrun",
+            "-p",
```

Comment:
Do we need a short flag version (-p)? I think having the full version is enough. We do not want to make it handy.